sdk/storage/azure-storage-queue/samples/queue_samples_authentication.py

```diff
@@ -49,7 +49,8 @@ def authentication_by_connection_string(self):
         # Instantiate a QueueServiceClient using a connection string
         # [START auth_from_connection_string]
         from azure.storage.queue import QueueServiceClient
-        queue_service = QueueServiceClient.from_connection_string(conn_str=self.connection_string)
+        if self.connection_string is not None:
```

Comment:
We should come up with a different strategy for if users don't specify the required environment variables. As is, this will sort of silently fail when we should probably throw if we don't have what we need.

We could do validate things at class construction time and save non-optional versions to the class but not all samples require all environment variables to be populated. So, I wonder if we just want to validate what we need at the top of each sample. Do some `if None: throw` checks. Open to other ideas.

This comment applies across all samples. We should come up with a pattern and apply it everywhere.

Reply:
We can't just check for the existence of all the env variables since some tests you aren't using all, so we should just assert at the top of each function.